client/src/templates/Challenges/components/challenge-description.css

```diff
@@ -2,13 +2,26 @@
   font-size: 16px;
 }
 
-.challenge-instructions blockquote {
+ .challenge-instructions blockquote, .challenge-instructions pre {
   background-color: #eee;
   color: #c7254e;
-  padding: 10px;
+  padding: 10px 10px 10px 15px;
   width: 100%;
   margin: 0;
   margin-bottom: 1.45rem;
   font-family: monospace;
   font-size: 16px;
-}
\ No newline at end of file
+  word-spacing: normal;
+  word-break: normal;
+  word-wrap: break-word;
+  border: unset;
+}
+
+.challenge-instructions pre code {
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Comment:
IMO, It's more common, do not break code lines.

Reply:
So what is your actual code change suggestion for this comment?

Reply:
do not change `white-space`, `word-break`, `word-wrap`

Reply:
But then it will not function the same as the current `blockquote` which is the whole point of this PR.

Reply:
If you decide that code blocks should simulate a blockquote, then OK, otherwise why add it here and then delete it in another PR?